api/src/main/java/org/openmrs/module/fhir/api/reverse/OpenmrsPersonUtil.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.fhir.api.reverse;
+import ca.uhn.fhir.model.dstu2.composite.AddressDt;
+import ca.uhn.fhir.model.dstu2.composite.HumanNameDt;
+import ca.uhn.fhir.model.dstu2.valueset.AddressUseEnum;
+import ca.uhn.fhir.model.dstu2.valueset.AdministrativeGenderEnum;
+import ca.uhn.fhir.model.dstu2.valueset.NameUseEnum;
+import ca.uhn.fhir.model.primitive.IdDt;
+import ca.uhn.fhir.model.primitive.StringDt;
+import org.openmrs.Patient;
+import org.openmrs.Person;
+import org.openmrs.PersonAddress;
+import org.openmrs.PersonName;
+import org.openmrs.api.context.Context;
+import java.util.List;
+
+import static java.lang.String.valueOf;
+
+        public class OpenmrsPersonUtil {
+
+         public static Person generateOpenMRSPerson(ca.uhn.fhir.model.dstu2.resource.Person personFHIR) {
+
+                //Set Person ID
+                Person omrsPerson = new Person();
+                IdDt uuid = new IdDt();
+                uuid.setId(personFHIR.getId());
+                omrsPerson.setUuid(valueOf(uuid));
+
+                for (HumanNameDt humanNameDt : personFHIR.getName()) {
+                PersonName personName = new PersonName();
+                if (humanNameDt.getUse() != null) {
+                String getUse = humanNameDt.getUse();
+
+                 if (valueOf(NameUseEnum.USUAL) == getUse) {
```

Comment:
I see some formatting issues in if else statements please format them accordingly